app/controllers/users_controller.rb

```diff
@@ -1,34 +1,37 @@
 class UsersController < ApplicationController
 
-  include RestrictedController
-  before_filter :changeable_filter,
-    :only => [
-      :show,
-      :edit,
-      :update,
-      :reset_password
-    ]
-  before_filter :manager, :only => [:destroy]
-  before_filter :manager_or_researcher,
-    :only => [
-      :index,
-      :account_report
-    ]
   after_filter :store_location, :only => [:index]
 
-  def changeable_filter
-    @user = User.find(params[:id])
-    redirect_home unless @user.changeable?(current_visitor)
+  rescue_from Pundit::NotAuthorizedError, with: :pundit_user_not_authorized
+
+  private
+
+  def pundit_user_not_authorized(exception)
+    if exception.query.to_s == 'edit?'
+      flash[:warning] = "You need to be logged in first."
+      redirect_to login_url
+    else
+      flash[:notice] = "Please log in as an administrator"
+      redirect_to :home
+    end
   end
 
+  public
+
   def new
     #This method is called when a user tries to register as a member
+    # PUNDIT_REVIEW_AUTHORIZE
+    # authorize User
```

Comment:
creating a user does not need to be authorized.
you can use `skip_authorization` here to indicate no authorization is needed.